src/main/java/frc/robot/vision/VisionConstants.java

```diff
@@ -0,0 +1,21 @@
+package frc.robot.vision;
+
+import edu.wpi.first.apriltag.AprilTagFieldLayout;
+import edu.wpi.first.apriltag.AprilTagFields;
+
+public class VisionConstants {
+
+	public static final String FILTERED_DATA_LOGPATH_ADDITION = "FilteredData/";
+
+	public static final String NON_FILTERED_DATA_LOGPATH_ADDITION = "NonFilteredData/";
+
+	public static final String SOURCE_LOGPATH_ADDITION = "VisionSource/";
+
+
+	public static final int LIMELIGHT_ENTRY_ARRAY_LENGTH = 6;
+
+	public static final int NO_APRILTAG_ID = -1;
+
+	public static final AprilTagFieldLayout APRIL_TAG_FIELD_LAYOUT = AprilTagFieldLayout.loadField(AprilTagFields.k2024Crescendo);
```

Comment:
nah the template should not have the apriltag field of 2024

Reply:
I have to have something in there :/
null is fine?